.github/dependabot.yml

```diff
@@ -1,12 +1,11 @@
 version: 2
 updates:
-- package-ecosystem: npm
-  directory: "/"
-  schedule:
-    interval: weekly
-  open-pull-requests-limit: 10
-  versioning-strategy: widen
-  ignore:
-  - dependency-name: mocha
-    versions:
-    - ">= 8.1.a, < 8.2"
```

Comment:
I can't see how this would still be needed, this repo is using mocha v10

Reply:
Haha indeed.